tools/Scripts/MakeInstaller.py

```diff
@@ -243,7 +243,7 @@ def installQtInstallerFramework():
             installer=qtifwSetupExe(),
             silent_script=silent_script
         )
-        time.sleep(10)
+        time.sleep(30)
```

Comment:
I thought the installer was being generated already without a problem - why change the sleep here?

Reply:
It wasn't necessary. Went back to the old value.